File: dart/lib/BluefinDecryptxError.dart
class BluefinDecryptxError extends Error {
  final bool isBluefinDecryptxError = true;

  final String sdk = 'BluefinDecryptx';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinDecryptxError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinDecryptxError: ' + code + ': ' + message;
}

File: ts/src/BluefinDecryptxError.ts

import { Context } from './Context'


class BluefinDecryptxError extends Error {

  isBluefinDecryptxError = true

  sdk = 'BluefinDecryptx'

  code: string
  ctx: Context

  constructor(code: string, msg: string, ctx: Context) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

export {
  BluefinDecryptxError
}

